cmd/krel/cmd/push.go

```diff
@@ -76,6 +80,68 @@ var pushBuildCmd = &cobra.Command{
 	},
 }
 
+type stageFile struct {
+	srcPath  string
+	dstPath  string
+	required bool
+}
+
+var gcpStageFiles = []stageFile{
+	{
+		srcPath:  filepath.Join(release.GCEPath, "configure-vm.sh"),
```

Comment:
Non-blocking: Do we really even need this file?

Check to see if it exists in `master`, `release-1.1{5..7}` branches. 
If it does, set required to `true`. If it doesn't, remove it.